docs/quickstart.rst

```diff
@@ -114,6 +114,27 @@ dimensions at native magnification or at a specified level:
    Dimensions at level 1: (7500, 8496)
    Dimensions at level 2: (1875, 2124)
 
+.. note::
+    If the native magnification, *i.e.* the magnification factor used to scan the slide, is provided in the slide properties, it is also possible
```

Comment:
```suggestion
    If the native magnification, *i.e.,* the magnification factor used to scan the slide, is provided in the slide properties, it is also possible
```

Reply:
not sure its needed

Reply:
it depends whether we are writing British or American. I guess we always used American english